pygi-marshal-from-py.c:747:8: error: implicit declaration of function 'strdup'

--- origsrc/pygobject-3.4.2/gi/pygi-marshal-from-py.c	2012-11-12 01:01:19.000000000 -0600
+++ src/pygobject-3.4.2/gi/pygi-marshal-from-py.c	2012-12-16 19:32:05.711570400 -0600
@@ -744,7 +744,7 @@ _pygi_marshal_from_py_unichar (PyGIInvok
 
        size = PyUnicode_GET_SIZE (py_arg);
        py_bytes = PyUnicode_AsUTF8String (py_arg);
-       string_ = strdup(PYGLIB_PyBytes_AsString (py_bytes));
+       string_ = g_strdup(PYGLIB_PyBytes_AsString (py_bytes));
        Py_DECREF (py_bytes);
 
 #if PY_VERSION_HEX < 0x03000000
